data/replaced_tags.txt

```diff
@@ -101,6 +101,8 @@ historic=cannon : historic=memorial
 historic=technical_monument : historic=memorial
 historic=shield : historic=memorial
 historic=war_memorial : historic=memorial
+historic=manor : castle_type=manor
+historic=fort : castle_type=fortress
```

Comment:
Not sure about this. This kind of replacement will not match tags into historic-castle-manor/fortress.